README.md

```diff
@@ -94,6 +96,17 @@ g stands for global installation. If you use -g later you can use the CLI in any
 
  1.We have to load recaptcha library in signup component.We have to use ng recaptcha tool for that. 
  2. We have to call google site verify api for recaptcha thresold value.
+ 
+
+ ### For Changepassword
```

Comment:
@Bithika-Redapple Good you have added documentation for this. The titles could be title casing though, e.g. 'Change Password'. Same for your other sections, "User Tax Details", "User Consent Details", "Marketing Preferences", ect. The README docs should read like standard writing as opposed to code.

But it is good that you have added the information there. We can go back through and edit later.